readthedocs/proxito/views/utils.py

```diff
@@ -56,3 +60,22 @@ def proxito_404_page_handler(
     )
     r.status_code = http_status
     return r
+
+
+def allow_readme_html_at_root_url():
```

Comment:
```suggestion
def allow_readme_html_as_index():
```

root is generally the root URL `/` only (no subfolders), but here README is just another index (`index.html` at any level).